tests/fused_attention_stablehlo_test.py

```diff
@@ -313,6 +308,7 @@ def test_sdpa_inference(self):
 
   @jtu.run_on_devices("cuda")
   def test_sdpa_var_seq(self):
+    self.skipTest("Skip before fixed.")
```

Comment:
Is there a bug we can reference here? If not, can you add more context on why the test is skipped to the message string, please?

Reply:
I don't have more info now as I just noticed that with cuDNN 9.2 this test fail with less than 1% value diff larger than 1e05.